pages/item/_id/index.vue

```diff
@@ -91,15 +91,15 @@
         <div v-if="narrators && narrators.length" class="truncate text-sm">
           <template v-for="(narrator, index) in narrators">
             <nuxt-link :key="narrator" :to="`/bookshelf/library?filter=narrators.${$encode(narrator)}`" class="underline">{{ narrator }}</nuxt-link
-            ><span :key="index" v-if="index < narrators.length - 1">,&nbsp;</span>
+            ><span :key="${narrator.id}-comma" v-if="index < narrators.length - 1">,&nbsp;</span>
```

Comment:
The reason `index` is used here is because `narrator` is a string. So `narrator.id` doesn't exist. It is the same for genres.
genres & narrators are just arrays of strings

Reply:
Ah, my bad!